views/components/Hero/Hero.css

```diff
@@ -0,0 +1,20 @@
+.HeroCard_container {
```

Comment:
IMO the block should be the component name for consistency throughout the app, so either `Hero` or change the name of the component to `HeroCard`.

Reply:
[9d7c4dc](https://github.com/AMP-SCZ/dpdash/pull/468/commits/9d7c4dc05024fa07956aaa578a84f844f590db07)